packages/core/src/components/modal/modal.scss

```diff
@@ -20,7 +20,7 @@ $modals: (
     lg: 100%,
   ),
   $screen-md: (
-    xs: 75%,
+    xs: 50%,
```

Comment:
Maybe not breaking but should be highlighted in the release. People may have content in the modal of a certain width and expect it to be 75% on md screens.

Reply:
Good input!

Reply:
Yep, it came as user reported issue so we will write about it in Announcement channel to give users heads-up.